aptos-move/aptos-e2e-comparison-testing/src/main.rs

```diff
@@ -0,0 +1,115 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use aptos_comparison_testing::{
+    prepare_aptos_packages, DataCollection, Execution, ExecutionMode, APTOS_COMMONS,
+};
+use aptos_rest_client::Client;
+use clap::{Parser, Subcommand};
+use std::path::PathBuf;
+use url::Url;
+
+const BATCH_SIZE: u64 = 100;
+
+#[derive(Subcommand)]
+pub enum Cmd {
+    /// Collect and dump the data
+    Dump {
+        /// Endpoint url to obtain the txn data,
+        /// e.g. `https://api.mainnet.aptoslabs.com/v1` for mainnet.
+        /// To avoid rate limiting, users need to apply for API key from `https://developers.aptoslabs.com/`
+        /// and set the env variable X_API_KEY using the obtained key
+        endpoint: String,
+        /// Path to the dumped data
+        output_path: Option<PathBuf>,
+        /// Do not dump txns that were discarded
+        #[clap(long, default_value_t = false)]
+        skip_discarded_txns: bool,
+        /// Do not dump publish txns
+        #[clap(long, default_value_t = false)]
+        skip_publish_txns: bool,
+        /// Collect txns regardless whether the source code is available
+        #[clap(long, default_value_t = false)]
+        skip_source_code_check: bool,
+        /// Dump the write set of txns
+        #[clap(long, default_value_t = false)]
+        dump_write_set: bool,
+    },
+    /// Execution of txns
+    Execute {
+        /// Path to the data
+        input_path: Option<PathBuf>,
```

Comment:
Can you try dump a few txns and have a testsuite for running those txns?

Reply:
Added example txn data and execution output